Websites/webkit.org/wp-content/themes/webkit/standards-positions.php

```diff
@@ -201,12 +210,20 @@ function updateStandardsPositionsTableRows() {
     );
     const filterTopic = document.getElementById("filter-topic").value;
     const filterPosition = !document.getElementById("filter-position").checked;
+    let isEvenRow = false;
     for (const row of document.querySelector("main").querySelector("tbody")
         .rows) {
-        row.toggleAttribute(
-            "hidden",
-            shouldRowBeHidden(row, filterTextRE, filterTopic, filterPosition)
+        const hide = shouldRowBeHidden(
+            row,
+            filterTextRE,
+            filterTopic,
+            filterPosition
         );
+        row.toggleAttribute("hidden", hide);
+        row.classList.toggle("even", !hide && isEvenRow);
+        if (!hide) {
+            isEvenRow = !isEvenRow;
+        }
```

Comment:
```suggestion
        if (!hide)
            isEvenRow = !isEvenRow;
```

This is WebKit style fwiw